### PR TITLE
Cause/Interests Tab Cypress Tests

### PR DESCRIPTION
**Pull request by lindsayrmaher**
### What's this PR do?

This pull request adds a test to make sure a users interests are correctly represented in their profile.

### How should this be reviewed?

👀 

### Any background context you want to provide?

Part of ongoing cypress cleanup.

### Relevant tickets

References [Pivotal #176438356](https://www.pivotaltracker.com/n/projects/2401401/stories/176438356).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [x] Added appropriate feature/unit tests.

### CODE REVIEW

File: cypress/integration/account-interests-tab.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          'ENVIRONMENT',
+          'LGBTQ_RIGHTS_EQUALITY',
+          'RACIAL_JUSTICE_EQUITY',
+        ],
+      },
+    });
+
+    // Log in & visit the interests tab:
+    cy.login(user);
+    cy.withFeatureFlags({ cause_preferences: true }).visit(
```
Comment (lindsayrmaher) [inline cypress/integration/account-interests-tab.js:27]: creating this test reminded me that we still have this tab behind a feature flag although i think it's available for all users on PROD...can we remove?

Comment (DFurnes) [reply cypress/integration/account-interests-tab.js:27]: I'd imagine so!

**Comment by lindsayrmaher** [comment]
@DFurnes this should be ready for another peek!

**Review by DFurnes — APPROVED** [review]
Quality cleanup! 🍃

**Pull request merged**